include/bluetooth/mesh/vnd/dm_common.h

```diff
@@ -21,7 +21,6 @@
 extern "C" {
 #endif
 
-#define BT_MESH_VENDOR_COMPANY_ID 0x0059
```

Comment:
I believe this should not be changed. We want to allow people to update/expand this model and use it for their own purpose and in that case, substitute this number with their own company ID. Applies to all changes in this file.